docs/source/user-guide/tasks/create-project-archive.rst

```diff
@@ -67,3 +67,43 @@ EXAMPLE::
         557  06-10-2016 10:33   iris/iris_plot/main.py
   ---------                     -------
        6003                     5 files
+
+Extracting the archive file
+===========================
+
+It is recommended that you unarchive Anaconda Project bundles using
+the ``unarchive`` command::
```

Comment:
I'd explain why. People need a motivation for something they would otherwise think they know how to do using their usual tools.

Reply:
Honestly, I've been using `anaconda-project` for many years and until this PR I didn't even know that `unarchive` existed. Allowing OSX to extract archives never caused me problems so unless there is something important I have been missing all along (which seems unlikely!) I consider `unarchive` as being included a command included for completeness. It is probably the easiest option in a CLI with no GUI but otherwise I'm not sure why it needs to be recommended as opposed to available when needed.